toolkit/src/errors.rs

```diff
@@ -7,7 +7,9 @@ use failure::Fail;
 #[derive(Debug, Fail)]
 #[fail(display = "error")]
 pub enum Error {
-    #[fail(display = "No bytes have been provided. Please use the --hex argument.")]
+    #[fail(
+        display = "No bytes have been provided. Please use the --hex or --from-solidity argument"
+    )]
     DataRequestNoBytes,
     #[fail(
         display = "The string provided in the --hex field is not a valid hexadecimal byte string: {}",
```

Comment:
Now this error can happen without using the `--hex` field, when using the `from-solidity` option

Reply:
This change was already covered in this PR, I don't know why GitHub didn't show it to you :shrug: